### ISSUE TITLE
Wacht Am Rhein campaign?

### ISSUE THREAD

**Issue by barsoom1717** [nicupavel/openpanzer#192]
You start with no prestige, you don't get any prestige after each turn (as in zero), and the American forces are way too reactive and overwhelming from the very start. It appears to me this scenario and campaign are unwinnable.

**Comment by salexgit** [comment]
Hi

Yes, me and Nicu have noticed the same "issue". Please note that we provided the campaign as it is without changing anything, but we have been considering to increase prestige values. If you have any suggestions which prestige values should be provided, your feedback would be highly appreciated.

**Comment by barsoom1717** [comment]
I would suggest 1000-5000 to start, and maybe 100-250 on each turn?  I was able to won the game, but only with a Victory with just a few turns until it would have been a Tactical Victory.  I ended up losing a number of units, particularly artillery.  It was almost impossible to get into "overrun" status, which I had gotten pretty good at in past campaigns.

**Comment by salexgit** [comment]
Hi

Well, 1000-5000 is a huge difference, can you be more precise in your recommendation? By the way, it would be also nice if you provide your feedback on how much additional prestige should be allocated for every new scenario.

Speaking of prestige value for taking victory hexes, I cannot recall this parameter to be controlled by the campaign settings. It is just hard-coded in the game and should be same for all the campaigns/scenarios.

**Comment by barsoom1717** [comment]
I chose 1000-5000 because I had seen both those values ib the beginning of the campaign.  I forget which campaign started with 1000, and which started with 5000. 




I think equally as important, maybe more so, is like in the previous campaigns you get 138-327 (that is what I remember the other campaigns doing) on each turn.  Currently you start with zero prestige and get zero after each turn.




As far as the victory hexes go, I re-ran that turn and it worked correctly that time.




Get Outlook for Android







On Sun, Jan 20, 2019 at 7:33 PM -0600, "salexgit" <notifications@github.com> wrote:












Hi


Well, 1000-5000 is a huge difference, can you be more precise in your recommendation? By the way, it would be also nice if you provide your feedback on how much additional prestige should be allocated for every new scenario.


Speaking of prestige value for taking victory hexes, I cannot recall this parameter to be controlled by the campaign settings. It is just hard-coded in the game and should be same for all the campaigns/scenarios.

**Comment by salexgit** [comment]
Hi all

1. Since I was the one who prepared this campaign to be portable to OpenPanzer, I will be more than happy to increase prestige values. Even though I did not play the campaign, I also noticed that there was not enough prestige.

2. I full agree with observations made Nicu. Germany had one of the most experienced military divisions in that battle, in particular LSSAH and DasReich. I do not want to diminish military skills of US troops, but German should be at least as experienced as US units, or maybe even more.

3. I did not check this aspect at all. I can cross-check German tank types and how many there should while adjusting German units experience level.

Regards, Alex
